classes/profile_table.php

```diff
@@ -97,11 +97,11 @@ public function col_scripttype(object $record): string {
      */
     public function col_request(object $record): string {
         if ($this->is_downloading()) {
-            return $record->request;
+            return $record->request . $record->pathinfo;
```

Comment:
I think the request columns should be:

1) visibly just the full url, as close as possible to what the user originally saw
2) except with the wwwroot removed
3) the linked url will be the full url
4) it should also include the params, the params column can be removed
5) the visible url may be truncated down to say 100 chars (look at html_to_text to help with this)

Reply:
It's probably worth having a helper somewhere to rebuild the url, we will use this in a few places

Reply:
You can also look at shorten_text (in lib/moodlelib.php), which is also available as a built in mustache helper.

Reply:
I've updated the main PR description with the changes - visuals included.